lib/jest-helpers.js

```diff
@@ -0,0 +1,40 @@
+'use strict'
+
+const nunjucks = require('nunjucks')
+const cheerio = require('cheerio')
+
+const COMPONENT_DIR = 'src/components'
+
+nunjucks.configure(COMPONENT_DIR, {
+  trimBlocks: true,
+  lstripBlocks: true
+})
+
+function render (name, params) {
+  const output = nunjucks.render(name + '/template.njk', { params })
+  const $ = cheerio.load(output)
+  return { output, $ }
+}
+
+const yaml = require('js-yaml')
```

Comment:
Is this conventional? I'd expect to see all includes at the top of the file.

Reply:
I was grouping with where they're used, agree lets move these 👍